.gitignore

```diff
@@ -1,2 +1,3 @@
 .cm
 .smackage
+bin
```

Comment:
Ah, yeah that's fine. I just wanted to ignore the heapimage thing that gets generated.